packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -49,44 +57,86 @@ contract AnchorStateRegistry is Initializable, ISemver {
     /// @param game Game that was used as the new anchor game.
     event AnchorUpdated(IFaultDisputeGame indexed game);
 
-    /// @notice Thrown when an unauthorized caller attempts to set the anchor state.
-    error AnchorStateRegistry_Unauthorized();
+    /// @notice Emitted when the respected game type is set.
+    /// @param gameType The new respected game type.
+    event RespectedGameTypeSet(GameType gameType);
 
-    /// @notice Thrown when an invalid anchor game is provided.
-    error AnchorStateRegistry_InvalidAnchorGame();
+    /// @notice Emitted when the retirement timestamp is set.
+    /// @param timestamp The new retirement timestamp.
+    event RetirementTimestampSet(uint256 timestamp);
+
+    /// @notice Emitted when a dispute game is blacklisted.
+    /// @param disputeGame The dispute game that was blacklisted.
+    event DisputeGameBlacklisted(IDisputeGame indexed disputeGame);
 
     /// @notice Thrown when the anchor root is requested, but the anchor game is blacklisted.
     error AnchorStateRegistry_AnchorGameBlacklisted();
 
-    /// @notice Constructor to disable initializers.
-    constructor() {
+    /// @notice Thrown when an invalid anchor game is provided.
+    error AnchorStateRegistry_InvalidAnchorGame();
+
+    /// @notice Thrown when an unauthorized caller attempts to set the anchor state.
+    error AnchorStateRegistry_Unauthorized();
+
+    /// @param _disputeGameFinalityDelaySeconds The dispute game finality delay in seconds.
+    constructor(uint256 _disputeGameFinalityDelaySeconds) {
+        DISPUTE_GAME_FINALITY_DELAY_SECONDS = _disputeGameFinalityDelaySeconds;
         _disableInitializers();
     }
 
     /// @notice Initializes the contract.
     /// @param _superchainConfig The address of the SuperchainConfig contract.
     /// @param _disputeGameFactory The address of the DisputeGameFactory contract.
-    /// @param _portal The address of the OptimismPortal contract.
     /// @param _startingAnchorRoot The starting anchor root.
     function initialize(
         ISuperchainConfig _superchainConfig,
         IDisputeGameFactory _disputeGameFactory,
-        IOptimismPortal2 _portal,
-        OutputRoot memory _startingAnchorRoot
+        OutputRoot memory _startingAnchorRoot,
```

Comment:
Given how much we're changing in this PR anyway, would it be reasonable to rename this? it is in no way an `OutputRoot` - it's an `AnchorState` containing a hash (which is the output root) and l2BlockNumber.  If we look forward to interop, that would be a hash (super root this time) and timestamp, so ideally:

```
struct AnchorState {
  Hash root;
  uint256 sequenceNumber;
}
```

Not an issue if that isn't a simple rename though and maybe better to do in a separate PR anyway.

Reply:
Ok it's enough of a diff that I'll do it in a separate PR